index.html

```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
         </section>
 
         <section class="section-two">
-            <div class="section-title">
-                <h2>My Recent Works</h2>
-                <hr>
-            </div>
+          <div class="section-title">
+            <h2>My Recent Works</h2>
+            <hr>
+          </div>
         
-            <div class="recent-work">
-                <div class="image-1">
-                  <img src="./images/section-1.png" alt="">
-                </div>
-                <div class="section-one">
-                    <h2>Multi-Post Stories</h2>
-                    <p>
-                        A daily selection of privately personalized reads;
-                        no accounts or sign-ups required. 
-                        has been the industry's standard dummy text ever since the 1500s, 
-                        when an unknown printer took a standard dummy text.
-                    </p>                    
-                    <ul class="grid-container">
-                        <li class="grid-item1"><a href="#">css</a></li>
-                        <li class="grid-item2"><a href="#">html</a></li>
-                        <li class="grid-item3"><a href="#">bootstrap</a></li>
-                        <li class="grid-item4"><a href="#">Ruby</a></li>
-                    </ul>
-                    <button name="button" type="submit" class="btn-1">See Project</button>
-                </div>
+          <div class="recent-work">
+            <div class="image-1">
+              <img src="./images/section-1.png" alt="">
             </div>
-        
-            <div class="back-images">
-                <h2>Profesional Art Printing Data</h2>
+            <div class="section-one">
+              <h2>Multi-Post Stories</h2>
+              <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, 
+                when an unknown printer took a standard dummy.
+              </p>                    
+              <ul class="grid-container">
+                <li class="grid-item1"><a href="#">css</a></li>
+                <li class="grid-item2"><a href="#">html</a></li>
+                <li class="grid-item3"><a href="#">bootstrap</a></li>
+                <li class="grid-item4"><a href="#">Ruby</a></li>
```

Comment:
- [ ] Please make sure to align the navigation links exactly as Figma design i.e fix the distance between the links in Figma design there is more margin between the links

![image](https://user-images.githubusercontent.com/74744787/160025388-ecaa7a7a-f7ea-41f3-8c08-91eb4bb5d2ca.png)

![image](https://user-images.githubusercontent.com/74744787/160025482-049689c7-adf5-4552-90ca-57f1b7b0e471.png)